02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,108 @@
+html {
+    background-image: url("https://wallpaperscraft.com/image/paper_crumpled_light_shadow_35117_1920x1080.jpg");
```

Comment:
Have look at this: http://bit.ly/2ani2Ka

See the `.App-logo` and `@keyframes spin`
It's just pure CSS, that is how I'm animating that image (aca_circle_logo_no_text.png).

I'll do more research on the @keyframes Rule this weekend and will share my finding with you. I don't really understand it 100%, just know how to use it. That's how it starts. :smile: 
This is a good starting point for the both of us, http://www.w3schools.com/cssref/css3_pr_animation-keyframes.asp